shell/platform/darwin/ios/framework/Source/application_messages_impl.mm

```diff
@@ -12,7 +12,7 @@
 namespace {
 
 std::vector<char> SysNSStringToVector(NSString* string) {
-  if (!string.length)
+  if (!string || !string.length)
```

Comment:
`string.length` will return `0` if `string` is `nil` because you would end up sending the `length` selector to `nil` which would be zero. The `!string` check is redundant.